installer/manifests/common/templates/_images.tpl

```diff
@@ -0,0 +1,39 @@
+{{/*Return the proper serivce image name*/}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
+{{- define "common.images.image" -}}
+{{- $registryName := "" -}}
+{{- $repositoryName := .imageRoot.repository -}}
+{{- $tag := include "common.images.tag" (dict "imageRoot" .imageRoot "global" .global "defaultTag" .defaultTag) -}}
+{{- if .global }}
+    {{- if .global.registry }}
+     {{- $registryName = .global.registry -}}
+    {{- end -}}
+{{- end -}}
+{{- if .imageRoot.registry -}}
+  {{- $registryName = .imageRoot.registry -}}
+{{- end -}}
+{{- if $registryName }}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- else -}}
+{{- printf "%s:%s" $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{/*Return the proper serivce image tag*/}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
```

Comment:
left over?

Reply:
Do I miss sth? :-) this is the new func for getting the proper image tag

Reply:
ah now I get it ;-) jep it's a comment and it should tell the users how to use the func. You are right the should be 
`{{/*{{- include "common.images.tag" ... ` instead of image. Thx